src/transformers/models/trillsson_efficient/modeling_trillsson_efficient.py

```diff
@@ -0,0 +1,686 @@
+)
+class Trillsson_efficientModel(Trillsson_efficientPreTrainedModel):
+    def __init__(self, config: Trillsson_efficientConfig, add_pooling_layer: bool = True):
+        super().__init__(config)
+
+        self.config = config
+        # block efficientnetv3 info expand_ratio, output_filters, num_repeat, strides, use_se, conv_type
+        block_configs = [
+            [1, 24, 2, 1, 0, 1],
+            [4, 48, 4, 2, 0, 1],
+            [4, 64, 4, 2, 0, 1],
+            [4, 128, 6, 2, 1, 0],
+            [6, 160, 9, 1, 1, 0],
+            [6, 256, 15, 2, 1, 0],
+        ]
+        # building first layer
+        input_channel = make_divisible(
+            24 * self.config.depth_multiplier, self.config.depth_divisible_by, self.config.min_depth
+        )
+
+        self.stem = StemLayer(
+            config=self.config,
+            in_channels=1,
+            out_channels=input_channel,
+            stride=2,
+            norm_eps=self.config.norm_eps,
+            norm_momentum=self.config.norm_momentum,
+        )
+
+        # building inverted residual blocks
+        self.block = nn.ModuleList()
+        current_num_blocks = 0
+        total_blocks = float(sum(blocks_args[0] for blocks_args in block_configs))
+        for expand_ratio, output_filters, num_repeat, stride, use_se, conv_type in block_configs:
+            output_channel = make_divisible(
+                output_filters * config.depth_multiplier, self.config.depth_divisible_by, self.config.min_depth
+            )
+            conv_block = MBConvBlock if conv_type == 0 else FusedMBConvBlock
+            for i in range(num_repeat):
+                if i > 0:
+                    stride = 1
+                survival_probability = self.config.drop_connect_rate * current_num_blocks / total_blocks
+                self.block.append(
+                    conv_block(
+                        config=self.config,
+                        in_channels=input_channel,
+                        out_channels=output_channel,
+                        stride=stride,
+                        expand_ratio=expand_ratio,
+                        use_se=use_se,
+                        survival_probability=survival_probability,
+                        norm_eps=config.norm_eps,
+                        norm_momentum=config.norm_momentum,
+                    )
+                )
+                input_channel = output_channel
+                current_num_blocks += 1
+
+        # building last several layers
+        if config.depth_multiplier > 1.0:
+            output_channel = make_divisible(
+                1280 * config.depth_multiplier, config.depth_divisible_by, config.min_depth
+            )
+        else:
+            output_channel = 1280
+
+        self.top = TopLayer(
+            config=self.config,
+            in_channels=input_channel,
+            out_channels=output_channel,
+            norm_eps=self.config.norm_eps,
+            norm_momentum=self.config.norm_momentum,
```

Comment:
Same comment here, unless `TopLayer` is initialized with different values for `norm_eps` and need this kwargs, lets just remove the.